docs/meshstack.mesh-policies.md

```diff
@@ -0,0 +1,38 @@
+---
+id: meshstack.mesh-policies
+title: Mesh Policies
+---
+
+meshPolicies can be set up between meshObjects, i.e., meshCustomer, meshProject, meshLandingZone and
+meshCustomerUserGroup. In the administration area, there is an option of 'Policies' in the 'Compliance' section of the
+navigation area. After clicking policies, you can view all policies in the system. Since this is a BETA version we only
+support tags for policies between meshObjects.
+
+## Create Policies
+
+Policy creation is only possible for a user who has a partner admin role right. There is a 'Create Policy' button top
+right on the policies screen. After clicking on it, there is a redirection to the screen of Create Policy where all
+details need to be filled up. To create the policy, you need to add the policy name in the form. Then you need to add
+left and right meshObjects from the dropdown menu. For ease of use, we already restricted the object combinations which
```

Comment:
we dont call them left and right. Its simple one object and the other there is no direction

Reply:
yes, changed